.circleci/config.yml

```diff
@@ -135,7 +135,7 @@ jobs:
       - checkout
       - run: |
           echo 'export TIMEOUT=10m' >> $BASH_ENV
-          echo 'export CLUSTER_DEFINITION=examples/kubernetes.json' >> $BASH_ENV
+          echo 'export CLUSTER_DEFINITION=examples/kubernetes-vmss/kubernetes.json' >> $BASH_ENV
```

Comment:
`VMSS` is the default so updating the "k8s Linux default" to use vmss since all the other e2e tests use `AvailabilitySet`

Reply:
Instead of this, let's just update the `examples/kubernetes.json` api model to no longer specify an `"availabilityProfile"` (it should use vmss by default)